qemu/tests/qemu_io_blkdebug.py

```diff
@@ -110,15 +109,12 @@ def run(test, params, env):
                 params_sn, data_dir.get_data_dir(), image_sn)
             image_snapshot.remove()
 
-        error.context("Get error message from command perror", logging.info)
-        perror_cmd = "perror %s" % errn
-        std_msg = utils.system_output(perror_cmd)
-        std_msg = re.findall(re_std_msg, std_msg)
-        if std_msg:
-            std_msg = std_msg[0]
-        else:
+        error.context("Get error message", logging.info)
```

Comment:
Both perror and os.strerror() can return the error message for the giving error code. it is better to use strerror() directly, then we don't need to take care whether command perror is installed on the host.